profiles/multisite_drupal_communities/modules/custom/nexteuropa_ds_layouts

```diff
@@ -0,0 +1 @@
+../../../common/modules/custom/nexteuropa_ds_layouts
```

Comment:
the module nexteuropa_ds_layouts does not exist.

Reply:
Fixed